src/openapi.json

```
@@ -127,6 +127,30 @@
 				},
 				"required": ["id", "fileName", "sourceKey", "createdAt"]
 			},
+			"Organization": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "integer",
+						"readOnly": true,
+						"example": 3407
+					},
+					"employerIdentificationNumber": {
+						"type": "string",
+						"example": "111-11-1111"
```

Should be consistent with other examples (and expected EIN format).

```suggestion
						"example": "11-1111111"
```